config/routes.rb

```diff
@@ -10,5 +10,5 @@
   get '/auth/auth0/callback' => 'auth0#callback'
   get '/auth/failure' => 'auth0#failure'
   get 'dashboard' => 'dashboard#show'
-  delete 'logout' => 'logout#logout', via: :delete
+  get 'logout' => 'logout#logout', via: :destroy
```

Comment:
Updated this as it was giving out on production environment that [GET] /logout path did not exist